app/models/prepend/va_notify/appellant_notification.rb

```diff
@@ -35,9 +35,9 @@ def status
 
   class NoAppealError < StandardError; end
 
-  def self.handle_errors(appeal)
+  def self.handle_errors(appeal, template_name)
     fail NoAppealError if appeal.nil?
-    fail InactiveAppealError, appeal.external_id if !appeal.active?
+    fail InactiveAppealError, appeal.external_id if template_name == "Quarterly Notification" && !appeal.active?
```

Comment:
```suggestion
    fail InactiveAppealError, appeal.external_id if template_name == Constants.EVENT_TYPE_FILTERS.quarterly_notification && !appeal.active?
```
This may be overkill, but I am wondering if it'd be better to leverage the constant in case we ever change it.